bitnami/kube-state-metrics/values.yaml

```diff
@@ -8,11 +8,6 @@ global:
   #   - myRegistryKeySecretName
   # storageClass: myStorageClass
 
-  ## Additional labels to apply to all resources
-  ##
-  labels: {}
-  #   foo: bar
-
```

Comment:
Can you remove this from the README as well?

Reply:
Sure !